src/tk_scrollable_frame.py

```diff
@@ -0,0 +1,104 @@
+import tkinter as tk
+import platform
+
+
+# ************************
+# Scrollable Frame Class
+# ************************
+class ScrollFrame(tk.Frame):
+    """
+    This is based on the code from this GIST: ttps://gist.github.com/mp035/9f2027c3ef9172264532fcd6262f3b01
+
+    This class provides a scrollable frame as a nearly drop-in replacement for a tkframe.
+
+    Add items to the ScrollFrame.viewPort, rather than to the ScrollFrame.
+    ```
```

Comment:
Unfortunately the creator of this gist has not released it under a license 😞 It looks really cool and I definitely like the change, but we legally can't put this into the codebase because OpenMCR is distributed under the GPL license. [This article](https://choosealicense.com/no-permission/) has some more details:

> If you find software that doesn’t have a license, that generally means you have no permission from the creators of the software to use, modify, or share the software. Although a code host such as GitHub may allow you to view and fork the code, this does not imply that you are permitted to use, modify, or share the software for any purpose.

Instead, is there a package we could import through pip? Then we wouldn't have to worry about licensing. Or if we can find some GPL-licensed code then we could do that.

Reply:
I am contacting the author about the license. This was the most transparent bit of code I could find to do this.

Reply:
If you could convince them to place the GPL license in the gist, that would work great!

Reply:
I've had no response, but the present version of the GIST is a compilation of different people's suggestions. Some of the suggestions were public discussions on stackoverflow. I have tried prodding the author another way and hope to get a response, but as it incorporates publicly available code that has been freely exchanged and is being made publicly available, I think it is probably legally in the public domain. As such we can legally distribute it under a valid copyleft license such as GPL 3.

Reply:
Actually the source code of the GIST has been updated to include reference to the license. I will update the doc string to reflect that.

Reply:
Commit https://github.com/iansan5653/open-mcr/pull/74/commits/3d56f9a609c0f140b77f6726db0508220f96217f updates the docstring to include the license information.